### PR TITLE
Add hidden seeds from migrations

### PR DESCRIPTION
**Pull request by BravoSimone**
As requested in issue #1502 I'm adding seeds for data previously created only in migrations.
I've added only what I thought would be necessary, see the [comment](https://github.com/solidusio/solidus/issues/1502#issuecomment-303378552) for the complete list

### CODE REVIEW

**Comment by mamhoff** [comment]
When would these files be run?

**Comment by BravoSimone** [comment]
`seed.rb` requires every file into `db/default/` recursively so each time `rake db:seed` is used it will also load this data

File: core/db/default/spree/refund_reason.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+Spree::RefundReason.find_or_create_by(name: "Return processing")
```
Comment (kennyadsl) [inline core/db/default/spree/refund_reason.rb]: I think this should be added [here](https://github.com/solidusio/solidus/blob/4354667e613ceeac29c4fc30c7fbc1e5abc56083/core/db/default/spree/return_reasons.rb) instead

**Comment by mamhoff** [comment]
The `cart_tax_country_iso` determines for which country taxes are shown in the cart. As stores that sell to the US do not need to show taxes in the cart, I think it's fine to leave this as `nil`.

**Pull request merged**